tests/security/attributes/run_test.pl

```diff
@@ -59,47 +59,58 @@
   die("ERROR: Too many arguments passed");
 }
 
+sub gov_gen {
+    my $aup = shift;
+    my $ejac = shift;
+    my $dpk = shift;
+    my $lpk = shift;
+    my $rpk = shift;
+
+    system("./gov_gen -aup $aup -ejac $ejac -dpk $dpk -lpk $lpk -rpk $rpk -o governance.xml");
+    system("openssl smime -sign -in governance.xml -text -out governance.xml.p7s -signer $DDS_ROOT/tests/security/certs/permissions/permissions_ca_cert.pem -inkey $DDS_ROOT/tests/security/certs/permissions/permissions_ca_private_key.pem");
```

Comment:
I'm not sure we can guarantee that `openssl` is in the PATH on all builds.  What if we had `gov_gen` take care of this step by linking to the OpenSSL libs.